hedera-mirror-rest/__tests__/globalSetup.js

```diff
@@ -30,7 +30,17 @@ let dockerDb;
 
 const createDbContainer = async (maxWorkers) => {
   const image = isV2Schema() ? v2DatabaseImage : v1DatabaseImage;
-  dockerDb = await new PostgreSqlContainer(image).start();
+  const initSqlPath = path.resolve('../../hedera-mirror-node/hedera-mirror-common/src/test/resources/init.sql');
```

Comment:
Use path.join for cross platform compatibility

```suggestion
  const initSqlPath = path.join('..', '..', 'hedera-mirror-node', 'hedera-mirror-common', 'src', 'test', 'resources', 'init.sql');
```

Reply:
Thanks I have updated it to use path.join